core/iwasm/libraries/lib-socket/src/wasi/wasi_socket_ext.c

```diff
@@ -143,8 +143,9 @@ accept(int sockfd, struct sockaddr *addr, socklen_t *addrlen)
     error = __wasi_sock_accept(sockfd, 0, &new_sockfd);
     HANDLE_ERROR(error)
 
-    error = getpeername(new_sockfd, addr, addrlen);
-    HANDLE_ERROR(error)
+    if (getpeername(new_sockfd, addr, addrlen) == -1) { 
+        return -1; 
```

Comment:
@JamesMenetrey There are trailing spaces at the end of the two lines, which caused CI code guideline check error, could you please fix them? Thanks.

Reply:
Just trailed them :)